### ISSUE TITLE
Telegram desktop does not open links in browser (seccomp problem?)

### ISSUE THREAD

**Issue by clara2g** [netblue30/firejail#3031]
Hello :) fresh install of Fedora 31 with Firejail and noticed that clicking links in Telegram with Firejail does not open them in default browser (tried with Firefox, Falcon an Brave). I have found that commenting the seccomp line in telegram.profile fixes the issue.

**Comment by rusty-snake** [comment]
It should be enough to allow `chroot`, in firejail master you can simply add `!chroot` after `seccomp` (with one space).

**Comment by rusty-snake** [comment]
@clara2g 
I'm closing here due to inactivity, please fell free to reopen if you have more questions.

**Comment by Micha-Btz** [comment]
I'm facing the same problem. Can you please explain in which file I musst allos chroot?

**Comment by rusty-snake** [comment]
`echo 'seccomp !chroot' >> ~/.config/firejail/telegram.local` or
`sudo sed -i 's/seccomp/seccomp !chroot/' /etc/firejail/telegram.profile`

**Comment by Micha-Btz** [comment]
This I have tested, don't work.
Warning: an existing sandbox was detected. /usr/bin/firefox will run without any additional sandboxing features
Error: Access was denied while trying to open files in your profile directory. 
So I whitelisted ~/.mozilla and it works.

One strange thing is,if I change this under the defaul profile in /etc/firejail/telegram.profile it works.
If I create a telegram.local firle with
seccomp !chroot
noblacklist ${HOME}/.mozilla
something went wrong and telegram is not able to load it's configuration.
